src/Events/Integrations/Modules/Ticket_Emails/Emails.php

```diff
@@ -0,0 +1,221 @@
+<?php
+/**
+ * Class Emails.
+ *
+ * @since   TBD
+ *
+ * @package TEC\Events\Integrations\Modules\Ticket_Emails
+ */
+
+namespace TEC\Events\Integrations\Modules\Ticket_Emails;
+
+use TEC\Tickets\Emails\Email_Abstract;
+use Tribe__Events__Main as Main;
+use Tribe\Utils\Lazy_String;
+
+/**
+ * Class Emails.
+ *
+ * @since   TBD
+ *
+ * @package TEC\Events\Integrations\Modules\Ticket_Emails
+ */
+class Emails {
+	/**
+	 * Filters the placeholders for the Tickets Emails.
+	 *
+	 * @since TBD
+	 *
+	 * @param array<string,mixed> $placeholders The placeholders for the Tickets Emails.
+	 * @param string              $email_id     The email ID.
+	 * @param Email_Abstract      $email_class  The email class.
+	 *
+	 * @return array<string,mixed> The filtered placeholders for the Tickets Emails.
+	 */
+	public function filter_tec_tickets_emails_placeholders( $placeholders, $email_id, $email_class ) {
+		$post_id = $email_class->__get( 'post_id' );
```

Comment:
```suggestion
		$post_id = $email_class->post_id;
```
Why are we calling the magic method directly? Not an ideal approach, let's use the interface as intended.

Reply:
Not in this case, that will make it fail, as the `post_id` is set to the class `$data`. Please [see this](https://github.com/the-events-calendar/event-tickets/blob/release/G23.edamame/src/Tickets/Emails/Email_Abstract.php#L830).